index.html

```diff
@@ -44,4 +44,171 @@ <h1 class="headline-heading heading-large">I'm Mariane <br> Glad to see you!</h1
+  <!-- Headline Section Ends Here -->
+
+  <!-- Works Section Starts Here -->
+  
+  <section id="portfolio" class="projects">
+    <!-- Card 1 starts -->
+    <ul class="works-cards">
+      <li class="works-card">
+        <img src="./Img/snapshoot-portfolio.png" alt="Recent Work">
+        <div class="works-card-main">
+          <h3 class="works-card-heading-large">Tonic</h3>
+          <div class="works-card-client">
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
+          </div>
+          <p class="paragraph works-card-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="works-card-tags">
+            <li class="works-card-tag">html</li>
+            <li class="works-card-tag">css</li>
+            <li class="works-card-tag">javascript</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </li>
+      <!-- Card 1 Ends -->
+      <!-- Card 2 starts -->
+      <li class="works-card">
+        <img src="./Img/snapshoot-portfolio-1.png" alt="Recent Work">
+        <div class="works-card-main">
+          <h3 class="works-card-heading-large">Multi-Post Stories</h3>
+          <div class="works-card-client">
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
+          </div>
+          <p class="paragraph works-card-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="works-card-tags">
+            <li class="works-card-tag">html</li>
+            <li class="works-card-tag">css</li>
+            <li class="works-card-tag">javascript</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </li>
+      <!-- Card 2 Ends -->
+      <!-- Card 3 starts -->
+      <li class="works-card">
+        <img src="./Img/snapshoot-portfolio-2.png" alt="Recent Work">
+        <div class="works-card-main">
+          <h3 class="works-card-heading-large">Tonic</h3>
+          <div class="works-card-client">
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
```

Comment:
- [ ] You have used fontawsome `icon` with `<i>` tag but haven't linked the `fontawsome` in `<head>` tag of the HTML file. Thats why `<i>` tag is not working properly. Kindly fix that issue.
- You can consider follow this [article](https://www.w3schools.com/icons/fontawesome_icons_intro.asp).
- Or you can copy the images given in the template and use it using `img` tag, you don't need fontawesome then.